Marlin/Configuration.h

```diff
@@ -353,6 +354,34 @@
 #define THERMAL_PROTECTION_BED     // Enable thermal protection for the heated bed
 
 //===========================================================================
+//================== High Temperature Thermistor Support ====================
+//===========================================================================
+
+/**
+ * Thermistors able to support high temperature tend to have a hard time getting
+ * good readings at room and lower temperatures. This means HEATER_X_RAW_LO_TEMP
+ * will probably be caught when the heating element first turns on during the
+ * preheating process, which will trigger a min_temp_error as a safety measure
+ * and force stop everything.
+ * To circumvent this limitation, we allow for a preheat time (during which,
+ * min_temp_error won't be triggered) and add a min_temp buffer to handle
+ * aberrant readings.
+ */
+// if you want to enable this feature for your hot end(s) thermistor(s)
+// set values > 0 in the constants below
+
+// How many consecutive low temperature error can occur before a min_temp_error
+// is triggered. Shouldn't be more than 10.
+//#define MAX_CONSECUTIVE_LOW_TEMPERATURE_ERROR_ALLOWED 0
+
+// How many milliseconds a hot end will preheat before starting to check the 
+// temperature. This value should NOT be set to the time it takes the
+// hot end to reach the target temperature, but should be set to the time it 
+// takes to reach the minimum temperature your thermistor can read. The lower
+// the better/safer, and shouldn't need to be more than 30 seconds (30000)
+//#define MILLISECONDS_PREHEAT_TIME 0
+
+//===========================================================================
```

Comment:
Items added to `Configuration.h` will need to be added to all copies of `Configuration.h` in the project. The rest of them live in the `example_configurations` folder. Perhaps these settings are more suited to `Configuration_adv.h`…?